app/webpack/js/app/models/json_api.model.js

```diff
@@ -0,0 +1,33 @@
+import { isArray } from "@js/app/helpers";
+
+const matcher = (data, item) => data.id == item.id && data.type === item.type;
+
+export class JsonApiModel {
```

Comment:
Not sure about the name here.  This is a base class that can initialize a thing given data that is in the shape of a `jsonapi` response (with `data` and `included` keys).  I wanted a common constructor for the basic models and for the more complicated one (like `ArtPiece`) i can make easy overrides.